src/containers/UserPage/ListView.js

```diff
@@ -1,52 +1,89 @@
 import React from 'react';
 
-import { Box, Clock, Text, Heading, Button, Grid, Paragraph, Collapsible } from 'grommet';
-import { format } from 'date-fns';
+import { Bar } from 'grommet-icons';
+
+import { Heading } from 'grommet';
+import BigClock from '../../components/BigClock/index';
+
+import differenceInCalendarDays from 'date-fns/difference_in_calendar_days'
+
+import { Row, Container, Col, Card, CardTitle, CardText } from 'reactstrap';
+import { StyledCardBody } from './style';
 
 class ListView extends React.Component {
     render() {
         let {
             events,
-            openNotification,
             match: { url },
         } = this.props;
 
+        if (events.length === 0) {
+            return (
+                <div>
+                    There is no events
+                </div>
+            )
+        }
+
+        let closestEvent = events.sort((a,b) => a.starttime - b.starttime)[0];
```

Comment:
The events should be ordered from the database, so this sorting is probably unnecessary.

Reply:
Should be fixed now.